unlock-app/src/components/creator/CreatorLock.js

```diff
@@ -97,28 +69,20 @@ const CreatorLockRow = styled.div`
   color: var(--slate);
   padding: 10px 0 10px 0;
   height: 64px;
-`
-
-const CreatorLockSaved = styled.div`
   display: grid;
-  grid-template-columns: 1fr 2fr repeat(4, 1fr) 2fr;
+  ${gridLayout}
```

Comment:
So I think we can actually parametrize syled components which would allow you to include the logic from `gridLayout` in here.

Reply:
see "Adapting based on props" on [this page](https://www.styled-components.com/docs/basics).

Reply:
:+1: adjusted to use this.